mllib/src/main/scala/org/apache/spark/mllib/linalg/Matrices.scala

```diff
@@ -254,6 +255,7 @@ class DenseMatrix(
    * @param numRows number of rows
    * @param numCols number of columns
    * @param values matrix entries in column major
+   * @since 1.3.0
    */
   def this(numRows: Int, numCols: Int, values: Array[Double]) =
```

Comment:
Oh never mind, I could notice that from the diff view :( . Looks all right to me.